charts/prometheus-adapter/values.yaml

```diff
@@ -13,9 +13,6 @@ metricsRelistInterval: 1m
 
 listenPort: 6443
 
-# User to run adapter container as
-runAsUser: 10001
```

Comment:
even that the value does not change this might be not working for updating users, which change d that values.
therfore major version  update is needed with upgrade instructions in readme.

Reply:
Understood. Upgrade note added, major version number bumped.